build-logic/settings.gradle.kts

```diff
@@ -23,26 +23,16 @@ pluginManagement {
 dependencyResolutionManagement {
     repositories {
         maven {
-            name = "Gradle libs"
-            url = uri("https://repo.gradle.org/gradle/libs")
-            mavenContent {
-                // This repository contains an older version which has been overwritten in Central
-                excludeModule("com.google.j2objc", "j2objc-annotations")
+            name = "ge-release-candidates"
+            url = uri("https://repo.gradle.org/gradle/enterprise-libs-release-candidates")
+            content {
+                val rcAndMilestonesPattern = "\\d{1,2}?\\.\\d{1,2}?(\\.\\d{1,2}?)?-((rc-\\d{1,2}?)|(milestone-\\d{1,2}?))"
```

Comment:
❓ why do we need this version regex? What happens if we only filter by the group and artifactId?

Reply:
If we would not supply this regex we would query this repository for released versions of the enterprise and test distribution plugins although we know they can not be downloaded from there since releases are only available via the plugin portal.